src/components/Icon/Icon.stories.tsx

```diff
@@ -16,15 +22,28 @@ type Story = StoryObj<typeof Icon>;
 
 export const Base: Story = {
   name: "Base",
-
+  argTypes: {
+    name: {
+      options: Object.keys(ICONS),
+      control: { type: "select" },
+    },
+  },
   args: {
-    name: "facebook",
+    name: ICONS.facebook,
   },
 };
 
-export const Default: Story = {
-  render: () => <Icon name={ICONS.plus} />,
-  name: "Default",
+export const Standart: Story = {
```

Comment:
There is a typo in "standard". Please update all references to have "d" as the last letter.